src/snowflake/snowpark/functions.py

```diff
@@ -3013,6 +3015,9 @@ def udf(
             The source code is dynamically generated therefore it may not be identical to how the
             `func` is originally defined. The default is ``True``.
             If it is ``False``, source code will not be generated or displayed.
+        strict: Whether the created UDF is strict. A strict UDF will not invoke the UDF if any input is
+            null. Instead, a null value will always be returned for that row. Note that the UDF might
+            still return null for non-null inputs.
```

Comment:
Is the last sentence necessary?

Reply:
Same applied to UDTF and SP docstrings.

Reply:
This is copied from https://docs.snowflake.com/en/sql-reference/sql/create-function.html#optional-parameters, but I'm open to remove it :)

Reply:
I vote for removal then. @sfc-gh-aling What do you think?

Reply:
though I feel it's not needed, but I think it's a good practice to align with the doc for searchability, and being verbose with one extra sentence doesn't harm too much.

Reply:
I guess I'll just keep it? :)